lib/Db/Row2Mapper.php

```diff
@@ -0,0 +1,645 @@
+<?php
+
+namespace OCA\Tables\Db;
+
+use DateTime;
+use OCA\Tables\Errors\InternalError;
+use OCA\Tables\Errors\NotFoundError;
+use OCA\Tables\Helper\ColumnsHelper;
+use OCA\Tables\Helper\UserHelper;
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\AppFramework\Db\MultipleObjectsReturnedException;
+use OCP\AppFramework\Db\QBMapper;
+use OCP\DB\Exception;
+use OCP\DB\IResult;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\IDBConnection;
+use OCP\Server;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+use Psr\Log\LoggerInterface;
+
+class Row2Mapper {
```

Comment:
I know the naming is not fixed yet, but just for mentioning I'd probably rename the old ones to "LegacyRow..:" and keep just for the migration purpose and have the new ones then be just "Row..."